stonesoup/platform/base.py

```diff
@@ -1,21 +1,143 @@
 # -*- coding: utf-8 -*-
+from abc import abstractmethod, ABC
 
+from ..functions import cart2sphere, cart2pol, coerce_to_valid_mapping
+from ..types.array import StateVector
 from ..base import Base, Property
 from ..types.state import State
 from ..models.transition import TransitionModel
+import numpy as np
 
 
-class Platform(Base):
-    """Platform base class
+class Platform(Base, ABC):
+    state = Property(State, doc="The platform state at any given point")
+    mapping = Property(np.ndarray, doc="Mapping between platform position and state dims")
```

Comment:
I'd call this `position_mapping`, as may want different mappings for velocity, acceleration or other elements of the state.

Reply:
This is done and will be pushed